BuildingSync.xsd

```diff
@@ -5758,6 +5758,34 @@
           <xs:documentation>True if measure is recommended.</xs:documentation>
         </xs:annotation>
       </xs:element>
+      <xs:element name="HazardousMaterials" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>List of the hazardous materials determined to be present, either through observation or as reported by others, which could potentially have a significant impact on the measure cost estimates.</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="HazardousMaterial" minOccurs="0" maxOccurs="unbounded">
+              <xs:annotation>
+                <xs:documentation>Hazard types, defined by the Globally Harmonized System of Classification and Labelling of Chemicals (GHS).</xs:documentation>
```

Comment:
@macintoshpie -- also, use labeling not labelling (former is US spelling, latter is UK).

Reply:
https://www.osha.gov/dsg/hazcom/ghsguideoct05.pdf